shared/app/index.native.tsx

```diff
@@ -11,6 +11,8 @@ import {enableFreeze} from 'react-native-screens'
 import {setKeyboardUp} from '../styles/keyboard-state'
 import {setServiceDecoration} from '../common-adapters/markdown/react'
 import ServiceDecoration from '../common-adapters/markdown/service-decoration'
+import {useUnmountAll} from '../util/debug'
```

Comment:
add unmount helper to help w/ tracking down memory leaks, it unmounts the world. also cleanup and split some hooks